README.md

```diff
@@ -609,6 +626,56 @@ have been saved by the Primary.
 ```
 
 
+# Partial-Verification Secondaries
+The Secondaries described and employed above ran full verification, employing
+the full suite of TUF and Uptane security checks. Uptane provides for a
+less demanding alternative: partial-verification Secondaries. These perform
+a few checks instead of the full set, at a measured cost to security, allowing
+weaker ECUs to still operate as Uptane-conformant Secondaries.
+
+The distinction is defined
+[in Section 8.3 of the Implementation Specification](https://docs.google.com/document/d/1wjg3hl0iDLNh7jIRaHl3IXhwm0ssOtDje5NemyTBcaw/edit?pli=1#heading=h.22u629s8u37q).
+Briefly, partial verification entails one signature check to validate metadata,
+and one signature check whenever the minimum time is ratcheted forward (for
+metadata expiration purposes). Partial-verification Secondaries need to know
+the Director's Targets role public key and the Timeserver's public key.
+
+You can run a demo partial-verification Secondary like so:
+(Mind that you have [the Uptane services](#window-1-the-uptane-services) and
+[a Primary client](#window-2-the-primary-clients) both running).
+
+```python
+>>> import demo.demo_secondary as ds
+>>> ds.clean_slate(partial_verifying=True)
+>>> ds.update_cycle()
+```
+
+The same optional arguments apply for the demo partial-verification Secondary
+as for the full-verification version [above](#window-3-the-secondary-clients)
+(ECU id/serial, vehicle id/VIN, Primary port, etc.).
+
+
+The Secondary's update_cycle() call does the following:
```

Comment:
Does it help readability if the parts of the update_cycle() that differ between full-verification and partial-verification stand out (perhaps in bold)?